knife/knife.gemspec

```diff
@@ -15,9 +15,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.1.0"
 
-  s.add_dependency "chef-config", ">= #{Chef::Knife::VERSION.split(".").first}"
-  s.add_dependency "chef-utils", ">= #{Chef::Knife::VERSION.split(".").first}"
-  s.add_dependency "chef", ">= #{Chef::Knife::VERSION.split(".").first}"
+  s.add_dependency "chef-config", "~> 18.6" #">= #{Chef::Knife::VERSION.split(".").first}"
```

Comment:
same as before, need to remove commented code

Reply:
Done